document/chapters/02-background.tex

```diff
@@ -243,24 +255,102 @@ \section{CAD Geometry}
 
 \section{CAD-Based Monte Carlo Radiation Transport}
 
-The Direct Accelerated Geometry Monte Carlo (DAGMC) software toolkit enables
-MCRT directly on CAD geometries \cite{Tautges_2009}. DAGMC was developed at the
-University of Wisconsin - Madison and has been coupled with many Monte Carlo
-codes (see Table \ref{tab:dagmc_implementations}). DAGMC relies on
-Cubit\cite{Blacker_1994} (and its commercial counterpart,
-Trelis\cite{Trelis_2018}) for model importing, cleaning, and tessellation.
-
-More specifically, DAGMC provides robust particle tracking for the underlying
-physics kernels of various Monte Carlo codes (see Table
-\ref{tab:dagmc_implementations}. It accomplishes this by discretizing CAD
-surfaces into sets of triangles representing surface boundaries. Volumes are
-then defined by all sets of triangles which represent bounding surfaces of a
-given volume. This surface mesh and the geometric relationships between sets of
-mesh elements, also known as Meshsets, are stored in the Mesh Oriented dAtaBase
-(MOAB) \cite{Tautges_2004}. These relationships, which preserve the geometric
-topology, are stored in a hierarchical structure within MOAB, relating volumes
-to their surfaces, surfaces to curves, and curves to geometric vertices. For the
-purposes of this work, only the relationship between volumes and surfaces are of
+\subsection{Translation-Based Methods}
+
+One approach to CAD-Based MCRT is to leverage the native geometry capabilities
+of Monte Carlo codes by translating CAD geometries into CSG representations. In
+this approach, the fidelity of the geometric representation is limited to the
+geometric primitives available in the supported Monte Carlo code. Most codes
+support only first and second order surfaces, so higher-order surfaces of the
+CAD model must be approximated to accommodate the limited representation of the
+native code. As a result, many of these methods will simplify single volumes
+into smaller, less complex volumes able to be represented in CSG. This can be
+problematic when tally information related to CAD entities is desired - surface
+or volumetric fluxes for a single entity in the CAD model must now be mapped to
+several entities in the CSG representation. The lack of one-to-one correlation
+between CAD and CSG entities makes model manipulation difficult in that a change
+to the CAD model may result more or fewer CSG entities after translation, adding
+difficulty to tracking tally mapping. Several translation approaches have been
+developed oriented toward the Monte Carlo N-Particle (MCNP) input format
+developed at Los Alamos National Laboratory \cite{LANL_MCNP5_VOLIII}. A few of
+these tools are described below:
+
+\codecallout{MCAM}{The Monte Carlo Automatic Modeling (MCAM) tool was developed
```

Comment:
FWIW... I think this capability is now bundled into their SuperMC code/package...

Reply:
I'll make mention of that. Thanks!